lib/model/rwfolder.go

```diff
@@ -87,6 +89,8 @@ type rwFolder struct {
 	copiers      int
 	pullers      int
 	shortID      uint64
+	newVersion   protocol.Vector
```

Comment:
Ok I fixed the other thing (with the glob) but I'm honestly not sure what to do here.  What do I pull new and cur version from to get just this file if not from at this location?